Makefile

```diff
@@ -3,18 +3,19 @@ VERSION := $(shell python2 -c "import json,sys;print json.loads(sys.stdin.read()
 TMP := /tmp/$(NAME)
 
 xpi:
-	cfx xpi
-	mv $(NAME).xpi build/
+	jpm xpi
+	-rm build/
```

Comment:
the dash at the beginning ignores the return value for this line (if trying to remove a dir that doesn't exist it will prompt an error but will also keep running and jump to the next line)